src/charm.py

```diff
@@ -117,6 +125,28 @@ def __init__(self, *args: Any) -> None:
 
         self.endpoints_provider = HydraEndpointsProvider(self)
 
+        self.metrics_endpoint = MetricsEndpointProvider(
+            self,
+            relation_name=self._prometheus_scrape_relation_name,
+            jobs=[
+                {
+                    "metrics_path": "/metrics/prometheus",
+                    "static_configs": [
+                        {
+                            "targets": [f"*:{HYDRA_ADMIN_PORT}"],
+                        }
+                    ],
+                }
+            ],
+        )
+
+        self.loki_consumer = LogProxyConsumer(
+            self,
+            log_files=[self._log_path],
+            relation_name=self._loki_push_api_relation_name,
```

Comment:
similar to above - `logging` is the default loki relation name so there is no need to provide it
```suggestion

```

Reply:
I'm gonna copy paste my answer here, because this issue is raised in multiple places:

The default relation name for the class we use to interact with loki isn't logging but log-proxy.
You can check at https://github.com/canonical/loki-k8s-operator/blob/main/lib/charms/loki_k8s/v0/loki_push_api.py line 491.
The documentation and the communication around the default relation name has been a bit of a mess with contradictory information.
As a consequence of this, I decided to always include the relation name. I also think this is a good idea because the default relation names can be updated in the libs anytime without us knowing.